TF_Application/Assets/TouchFree_Application/Scenes/TouchlessOverlay.unity

```diff
@@ -233,6 +233,16 @@ PrefabInstance:
   m_Modification:
```

Comment:
All of these changes should come from the prefab and not from the TouchlessOverlay scene